nipap/nipap/smart_parsing.py

```diff
@@ -429,8 +433,6 @@ def _string_to_dictsql(self, part):
                     'val2': part[0]['word']
                     }
 
-        elif part.getName() == 'ipv4_address':
-            dictsql = {}
```

Comment:
No, it was never used. This is what I tried to describe above.

There are three formats involved here
 * text query
 * AST
 * dictsql

pyparsing parses a text query and outputs the AST (abstract syntax tree). Each element has a type, like 'ipv6_prefix'. The part you are looking at above is in the code that turns the AST into dictsql. For IPv4 we don't have a separate type but actually classify it as text / "else" and the classification of IPv4 or truly text is done in the ast to dictsql part.. thus this would never match and so I removed it.